src/run.rs

```diff
@@ -162,15 +164,20 @@ fn list_images(state: &mut State) -> io::Result<Vec<ImageInfo>> {
                     continue;
                 }
 
-                let tab_index = trimmed_line.find('\t').ok_or_else(|| {
-                    io::Error::new(io::ErrorKind::Other, "Failed to split image ID and date.")
-                })?;
-                let (image_id, date_str) = trimmed_line.split_at(tab_index);
-                images.push(ImageInfo {
-                    id: image_id.trim().to_owned(),
-                    parent_id: None, // This will be populated below.
-                    created_since_epoch: parse_docker_date(&date_str)?,
-                });
+                let image_parts = trimmed_line.split('\t').take(3).collect::<Vec<_>>();
```

Comment:
I'm thinking it might be safer to remove the `.take(3)`, so that the pattern match on the following line will fail (as it should) if there are more than 3 parts in the line.